target/linux/bcm27xx/patches-6.6/950-0547-ASOC-dwc-Improve-DMA-shutdown.patch

```diff
@@ -13,15 +13,6 @@ Signed-off-by: Phil Elwell <phil@raspberrypi.com>
 
 --- a/sound/soc/dwc/dwc-i2s.c
```

Comment:
Please squash this into the first commit, don't break stuff just to fix it later.

Reply:
The first commit is just a result of sed runs.
Thanks to the separation, you can see what exactly has changed.
The second commit removes what is no longer needed or requires updating due to changes.
Squashing will make it so no one will be able to verify it.

Reply:
Each individual commit needs to be correct and not break compilation. If the patches need rebuilding after running the whitespace changes, then these changes need to be in the same commit as the whitespace changes.

Reply:
Patches squashed. I left the commit split in bd796b56b1a8fac9120917530c81070cd0b52f2d and 48236d0b91eb949ed1f787c4019307d8c81eee60 for change analysis.